### PR TITLE
[W5.2e][T09-A1]Low Weng Fei

### PR DESCRIPTION
**Pull request by low5545**
UtilsTest.java: Add JUnit test driver for seedu.addressbook.common.Utils.isAnyNull() method

### CODE REVIEW

**Comment by low5545** [comment]
Ready for review

File: src/seedu/addressbook/data/person/Contact.java [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+    @Override
+    public String toString() {
+        return value;
+    }
+
+    @Override
+    public boolean equals(Object other) {
+        return other == this // short circuit if same object
+                || (other instanceof Contact // instanceof handles nulls
+                && this.value.equals(((Contact) other).value)); // state check
```
Comment (jeffryhartanto) [inline src/seedu/addressbook/data/person/Contact.java]: Think about what happens If I set the value of the email and name to be the same and call this method.

**Comment by jeffryhartanto** [comment]
@low5545 Some comments added. Please close the PR after reading comments.

File: src/seedu/addressbook/data/person/Contact.java (hunk 1 above)
Comment (low5545) [reply src/seedu/addressbook/data/person/Contact.java]: true

**Pull request closed without merging**